### PR TITLE
Adds items to vim plugins list

### PR DESCRIPTION
**Pull request by marksteve**

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -8,7 +8,11 @@
 
 ## Vim plugins
 
-- ...
+- [syntastic](https://github.com/scrooloose/syntastic) - syntax checking plugin
```
Comment (rstacruz) [inline tools/README.md]: +0 for me, i find syntastic doesn't work well with neovim haha

Comment (rstacruz) [inline tools/README.md]: +1 on everything else tho

Comment (marksteve) [inline tools/README.md]: what do you use instead? i haven't switched to neovim because of the plugins i use :(

Comment (rstacruz) [inline tools/README.md]: one downside lang to syntastic: ruby is on by default, ruby is slow, therefore vim editing ruby gets slow (unless you turn it off yourself). though haven't used it in a while, so i'm not sure if this is still the case :)

Comment (victorsolis) [inline tools/README.md]: Paki lagyan na lang siguro ng sections, para madaling dagdagan ng alternatives.

And mag `nvim` ka na @marksteve. Onti lang tatamaan, kung meron.

Comment (marksteve) [inline tools/README.md]: @victorsolis Just did!
@rstacruz Ok yung neomake. Supported yung mga ~~checkers~~makers ko dati.

**Pull request merged**